src/main.cpp

```diff
@@ -1,9 +1,18 @@
 #include <Arduino.h>
+#include "Led.h"
 
-void setup() {
-  // put your setup code here, to run once:
+// Define output pin and period of blink
+#define Pinout 1
+#define led_delay 50 // This value x2 is the period in ms
```

Comment:
If possible, make these 'normal' variables and declare them in setup()?

Reply:
The setup() function is used to initialize variables and define pin mode. When you define them as normal variables, they won't be in the scope of loop()